config/manager/manager.yaml

```diff
@@ -17,7 +17,7 @@ spec:
         control-plane: capz-controller-manager
         aadpodidbinding: capz-controller-aadpodidentity-selector
       annotations:
-        kubectl.kubernetes.io/default-logs-container: manager
+        kubectl.kubernetes.io/default-container: manager
```

Comment:
heads-up: #2531 should merge soon, so reminder to rebase on top of main after that (which should remove this diff)